### ISSUE TITLE
Generating a segment-based TXTP with the segments in the wrong order?

### ISSUE THREAD

**Issue by apocalyptech** [bnnm/wwiser#6]
Hello!  I've got a `.bnk` file for Borderlands 3 which generates a bunch of TXTPs.  At least one of the TXTPs in question is a multi-segment file which plays a bunch of `.wem`s in order, but Wwiser's putting them in the wrong order.  As an example, here's the original TXTP as-generated by wwiser:

~~https://apocalyptech.com/scratchpad/wwiser/Oak_VO_Radio_Bunkers_and_Badasses_01%20%7br%7d.txtp~~ *(links taken down)*

... and here's a fixed version with the `.wem`s in the correct order (also note that the single random-group element occurs *twice* throughout the TXTP, rather than once as in the originally-generated version):

~~https://apocalyptech.com/scratchpad/wwiser/Oak_VO_Radio_Bunkers_and_Badasses_01-fixed.txtp~~

The `.bnk` in question, and all the `.wem` files necessary to play that particular TXTP file, can be downloaded here: ~~https://apocalyptech.com/scratchpad/wwiser/wem_and_bnk.zip~~

You'd want to use https://raw.githubusercontent.com/bnnm/wwiser-utils/master/wwnames/Borderlands%203%20(PC).txt for `wwnames.txt` for this one.

Let me know if I can provide any more info, etc!

**Comment by bnnm** [comment]
Should work now: [wwiser.zip](https://github.com/bnnm/wwiser/files/6977436/wwiser.zip)

**Comment by apocalyptech** [comment]
Nice, yep, works great!

FYI, when regenerating the whole set, I *do* get this in the output now:

    generator: ranseq without playlist  929492043 CAkRanSeqCntr
    generator: ranseq without playlist  148722588 CAkRanSeqCntr

I see that you've labelled that in the code as "remnant code, probably safe to remove," so I figure I should mention that it does seem to happen in BNKs from the wild.  The two in question in this case are both just real short sound effects.  Did you want the BNKs/WEMs for those, anyway, to look at?

**Comment by apocalyptech** [comment]
Well, if you *did* want to look at those couple of instances where that notice is being printed, I went ahead and zipped 'em up -- just a single `.bnk` and `.txtp` for each:

https://apocalyptech.com/scratchpad/wwiser/148722588.zip
https://apocalyptech.com/scratchpad/wwiser/929492043.zip

This isn't actually affecting me at all; I just figured you might find it useful, given the comments in the code.  I'd be happy to just close out this Issue.  :)  Cheers!
